Gemfile

```diff
@@ -20,23 +20,17 @@ gem 'bootstrap', '~> 4.6.0'
 gem 'bootstrap-icons-helper'
 gem 'jquery-rails'
 
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-# gem 'jbuilder', '~> 2.7'
 # Use Redis adapter to run Action Cable in production
-gem 'redis', '~> 4.0'
+gem 'redis', '~> 5.0'
 # Use Active Model has_secure_password
-gem 'bcrypt', '~> 3.1.7'
+gem 'bcrypt', '~> 3.1'
```

Comment:
This makes the requirement more lax, meaning installing every `3.x` version (e.g. `3.18.9`) will be fine. That might not be the case though. Since the original requirement probably came with the original `Gemfile`, it would suggest keeping it that way.